src/Dropbox.elm

```diff
@@ -80,6 +80,7 @@ import Json.Decode.Pipeline as Pipeline
 import Json.Encode
 import Maybe.Extra
 import Navigation
+import Platform.Sub
```

Comment:
I think this can be removed; `import Platform.Sub as Sub` is one of the default imports.

Reply:
Thanks, that's much better.